sklearn/svm/tests/test_svm.py

```diff
@@ -505,27 +505,37 @@ def test_bad_input():
 
 
 def test_unicode_kernel():
-    # Test that a unicode kernel name does not cause a TypeError on clf.fit
+    # Test that a unicode kernel name does not cause a TypeError
     if six.PY2:
         # Test unicode (same as str on python3)
-        clf = svm.SVC(kernel=unicode('linear'))
+        clf = svm.SVC(kernel=unicode('linear'), probability=True)
         clf.fit(X, Y)
+        clf.predict(T)
+        clf.predict_proba(T)
```

Comment:
decision function?

Reply:
yep sounds a good idea, works on my local copy but just for completeness